src/App.tsx

```diff
@@ -194,131 +184,142 @@ function App() {
-                                    Place a Pixel
-                                </button>
-                            </div>
-
-
-
-                            {/* <div className={styles.apps} style={{left: zoombasedAdjustment}}>
-                                <Apps
-                                    appStore={appStore}
-                                />
-                            </div> */}
-                        </>
-                    }/>
-                    <Route path="/governance" element={<Governance />} />
-                    <Route path="/new-proposal" element={<NewProposal />} />
-                    <Route path="/proposal/:id" element={<ProposalDetails />} />
+                                    {isProposalListVisible && (
+                                        <div className='mb-4'>
+                                            <div className=''>
+                                                <ProposalListForMain
+                                                    headerHeight={64}
+                                                    statusFilter={statusFilter}
+                                                />
+                                            </div>
+                                            <div className='pt-4'>
+                                                <NewProposalPopupForMain />
+                                            </div>
+                                        </div>
+                                    )}
+                                </div>
 
+                                <div className={styles.buttonContainer}>
+                                    <button
+                                        className={styles.placePixelButton}
+                                        onClick={() => {
+                                            toggleColorPicker();
+                                            setSelectedApp('p_war');
+                                        }}
+                                        style={{ display: isColorPickerVisible ? 'none' : 'flex' }}
+                                    >
+                                        Place a Pixel
+                                    </button>
+                                </div>
+                            </>
+                        }
+                    />
+                    <Route path='/governance' element={<Governance />} />
+                    <Route path='/new-proposal' element={<NewProposal />} />
+                    <Route path='/proposal/:id' element={<ProposalDetails />} />
                 </Routes>
             </div>
             <ToastContainer
-                position="top-right"
-                className={'w-full max-w-[420px] min-w-[200px] z-999 top-12'}
+                position='top-right'
+                className={'top-12 z-50 w-full min-w-[200px] max-w-[420px]'}
```

Comment:
z-50 : toast is behind of Create A New Proposal Popup.
z-999 might be too big, but 50 is not enough.

Reply:
Okey, I added custom zIndex `z-999` so that toast properly shown!